### DISCUSSION TITLE
I have configured domain correctly according to Vercel but my website wont show up

### DISCUSSION THREAD

**Discussion by digital-key-co** [vercel/community#4796] [Help]
### Summary

I have a next.js application on Vercel and i have added a domain to it which is from cloudflare, according to Vercel dashboard project domains i have done it right, but the application wont show up

This is cloudflare DNS records:
![image](https://github.com/vercel/community/assets/143458029/7c274e90-4967-472b-b5b0-d93ea967cd66)

This is Vercel dashboard project domains:
![image](https://github.com/vercel/community/assets/143458029/1e8cb087-d81c-4b4f-af19-ed33bcf0d10c)

The default vercel domain works fine.
I have used Vercel with Cloudflare but have never encountered this

### Steps to Reproduce

This is the error i get with no subdomain:
![image](https://github.com/vercel/community/assets/143458029/453cf2e9-96ee-4487-984c-864e672f47c4)

The www successfully redirects to no subdomain

**Comment by alphaBiota** [comment]
Cloudflare / Vercel Redirect issue. I had the same problem but found the solution documented here: https://vercel.com/guides/resolve-err-too-many-redirects-when-using-cloudflare-proxy-with-vercel

**Reply by digital-key-co** [reply]
Thank you.
as I said in the post `I have used Vercel with Cloudflare but have never encountered this`, the problem was I missed that part

**Comment by digital-key-co** [comment]
The solution is to use `Full (strict)` option in Cloudflare `SSL/TLS` section.
Here is the full article [How do I resolve "err_too_many_redirects" when using a Cloudflare proxy with Vercel?](https://vercel.com/guides/resolve-err-too-many-redirects-when-using-cloudflare-proxy-with-vercel)
